### PR TITLE
#1961 

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
Searching with F1 or Ctrl+F1 doesn't fire when caret is at the end of a word.

Also switching off couple of tests until I figure out proper mocking behavior of the snippet session.

### CODE REVIEW

File: src/Mocks/VisualStudio/VsEnumStreamMarkersMock.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        }
+
+        public int GetCount(out int pcMarkers) {
+            pcMarkers = _markers.Length;
+            return VSConstants.S_OK;
+        }
+
+        public int Next(out IVsTextStreamMarker ppRetval) {
+            ppRetval = null;
+            if (_index >= 0 && _index < _markers.Length) {
```
Comment (AlexanderSher) [inline src/Mocks/VisualStudio/VsEnumStreamMarkersMock.cs]: Is it possible that `_index` is less than 0?

Comment (MikhailArkhipov) [inline src/Mocks/VisualStudio/VsEnumStreamMarkersMock.cs]: Not really

Comment (MikhailArkhipov) [inline src/Mocks/VisualStudio/VsEnumStreamMarkersMock.cs]: Unless it overflows somehow

**Comment by AlexanderSher** [comment]
:+1:

**Pull request merged**